tls_e2e_test.go

```diff
@@ -67,7 +67,7 @@ func TestIntegrationTestnetTls(t *testing.T) {
 	ledger, _ := LedgerIDFromNetworkName(NetworkNameTestnet)
 	client.SetTransportSecurity(true)
 	client.SetLedgerID(*ledger)
-	client.SetMirrorNetwork([]string{"hcs.testnet.mirrornode.hedera.com:5600"})
+	client.SetMirrorNetwork([]string{"testnet.mirrornode.hedera.com:5600"})
```

Comment:
This is not a valid endpoint. Needs to be 443. Did you run this test?

Similar comment on other tests.

Reply:
You are correct. Yes, all tests passed. This test isn't exactly test if this is valid network, it reverts to `443`. I'll revert this commit and open another PR as it would require a lot more changes since the SDK expects that there are 2 endpoints. Thanks for noticing.